docs/what_is_data_science/How you can learn Data Science.md

```diff
@@ -0,0 +1,14 @@
+# How you can learn Data Science
+
+Data Science isn't just for Data Scientists! As a profession, we're passionate about sharing these skills and techniques.
+
+The [NHS Data Science community](https://data-science-community.analystx.uk/) is the home of spreading data science knowledge within the NHS.
+
+You can also learn lot about data science by simply getting to know the wider cross-govt/health community:
```

Comment:
*cross-government rather than cross-govt - looks better and is more readable

Reply:
Changed.